### ISSUE TITLE
docker-ce repo not installing on CentOS7/x64

### ISSUE THREAD

**Issue by sxa** [AdoptOpenJDK/openjdk-infrastructure#1641]
From [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/OS=CentOS7,label=vagrant/):
```
20:48:44 TASK [Add Docker Repo for CentOS 7] ********************************************
20:48:45 fatal: [127.0.0.1]: FAILED! => {"changed": true, "checksum_dest": null, "checksum_src": "cccda062c436669011790aa0d4da2514c80db74f", "dest": "/etc/yum.repos.d/docker-ce.repo", "elapsed": 0, "msg": "The checksum for /etc/yum.repos.d/docker-ce.repo did not match 6650718e0fe5202ae7618521f695d43a8bc051c539d7570f0edbfa5b4916f218; it was 8ab5599eef0afcac10cbd3e8670873efee20fcceb5fb3526a62edeade603cec7.", "src": "/home/vagrant/.ansible/tmp/ansible-tmp-1603223324.17-27406-79615042759019/tmp63lozY", "url": "https://download.docker.com/linux/centos/docker-ce.repo"}
20:48:45 
20:48:45 PLAY RECAP *********************************************************************
20:48:45 127.0.0.1                  : ok=70   changed=44   unreachable=0    failed=1    skipped=52   rescued=0    ignored=5   
```

**Comment by Haroon-Khel** [comment]
Using a quick online checksum checker, the checksum has changed to `8ab5599eef0afcac10cbd3e8670873efee20fcceb5fb3526a62edeade603cec7`

**Comment by Willsparker** [comment]
(For future reference, you can check the checksum by running `sha256sum <file>` )

**Comment by Willsparker** [comment]
Fixed via #1642
